Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,60 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import subprocess
+import re
+
+def hasPermissionsToAdmin():
+  pass 
+
+def executeRabbitmqctl(arg, *argv):
+  command =['sudo','/usr/sbin/rabbitmqctl','-q', arg] + list(argv)
+  res = subprocess.Popen(command, stdout = subprocess.PIPE)
+  cmd_out, cmd_err = res.communicate()
+  return cmd_out 
+
+def addUserWithoutPassword(user):
+  addUser(user)
+  clearUserPassword(user)
+def addUser(user, password = 'password'):
+  ret = executeRabbitmqctl('add_user', user, password) 
```

Comment:
The password should not be passed in, rather it should be taken from the (local) CS